frontend/src/features/apartment/ApartmentListPage.tsx

```diff
@@ -33,7 +33,7 @@ const ApartmentListItem = ({
 }: ApartmentListItemProps): JSX.Element => {
     // Combine ownerships into a single formatted string
     const ownershipsString = ownerships
-        .map((o) => `${o.owner.last_name} ${o.owner.first_name} (${o.owner.social_security_number})`)
+        .map((o) => `${o.owner.name} (${o.owner.identifier})`)
         .join(", ");
```

Comment:
Can you run `yarn run prettier-format` to fix the formatting here, as the variable names are now shorter, prettier wants it to be single-row and raises an issue about it :)

Reply:
done